### PR TITLE
 minor fixes

### PR DESCRIPTION
**Pull request by suhasa010**
- format changes
- extract URLs from text
- ignore via_bot updates in PM

### CODE REVIEW

File: src/telegram/bot.service.ts [hunk 1]
```diff
@@ -57,7 +58,11 @@ export class BotService {
 
     this.bot.on('inline_query', (ctx) => {
       if (ctx.inlineQuery.query != '') {
-        this.resolve(ctx.inlineQuery.query)
+        // Extract only the URL from the pasted text
+        let queryParams = ctx.inlineQuery.query.split('http', 2)
```
Comment (pirasalbe) [inline src/telegram/bot.service.ts:62]: This won't work with 
`text [amazon_url] text`

File: src/telegram/bot.service.ts [hunk 2]
```diff
@@ -57,7 +58,11 @@ export class BotService {
...
+        let extractedUrl = 'http' + queryParams[1]
```
Comment (pirasalbe) [inline src/telegram/bot.service.ts:63]: If you do this it should always work
`queryParams[1].split(' ')[0]`

File: src/telegram/bot.service.ts (hunk 1 above)
Comment (suhasa010) [reply src/telegram/bot.service.ts:62]: ah :(

**Comment by pirasalbe** [comment]
Merged into #12

**Pull request closed without merging**